### ISSUE TITLE
Some non-text doesn't meet the minimum contrast ratio requirement

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#764]
Samsung Galaxy J3 Prime
Android version 7.0
Application version: 1.0 (53)

Scenario to reproduce

Non-text that required to understand the content such as close button for COVID Alert menu, circles used to indicate number of steps on “How it works” carousel as well as focused state for some buttons on the COVID Alert Menu don’t meet contrast ratio requirement against adjacent colours.

Expected result
Non-text that that required to understand the content as well as the focused state for the buttons should have a contrast ratio of at least 3:1 against their adjacent colours.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

Screenshots
![contrast](https://user-images.githubusercontent.com/12256067/87823068-a29ece00-c840-11ea-86bf-ea8e2f9cf6cc.gif)
![contrast3](https://user-images.githubusercontent.com/12256067/87823072-a3cffb00-c840-11ea-9b8a-fbf17ef23f28.gif)
![contrast2](https://user-images.githubusercontent.com/12256067/87823076-a599be80-c840-11ea-9fd0-68930f7dba57.gif)

**Comment by JuliannaR** [comment]
@amazingphilippe can you weigh in here on the arrow. I'm not sure this is an issue because its recognized as a button, the circles change for me and the screen being shown renders with a black circle. Are we able to change the colour of the arrow indicator?
